frontend/src/cards/DisparityBarChartCard.tsx

```diff
@@ -137,7 +139,7 @@ function DisparityBarChartCardWithKey(props: DisparityBarChartCardProps) {
             ) : (
               <CardContent>
                 <MissingDataAlert
-                  dataName={metricConfig.chartTitle || metricConfig.shortLabel}
+                  dataName={metricConfig.chartTitleLines.join(" ")}
```

Comment:
like here, couldn't you use "chartTitleOneLine" or whatever you had created above for the filename ?